libraries/commerce/product/selectors/product.js

```diff
@@ -104,14 +104,16 @@ export const getCurrentProduct = createSelector(
  * @param {Object} state The application state.
  * @returns {number|null}
  */
-export const getProductBasePrice = (state) => {
-  const currentProduct = getCurrentProduct(state);
-  if (!currentProduct) {
-    return null;
-  }
+export const getProductBasePrice = createSelector(
```

Comment:
Please rename to `getProductUnitPrice`. Can be confusing considering that we have base products.